src/main/kotlin/app/revanced/patches/youtube/misc/fix/playback/patch/SpoofSignatureVerificationPatch.kt

```diff
@@ -82,6 +82,9 @@ class SpoofSignatureVerificationPatch : BytecodePatch(
 
         } ?: return OpenCronetDataSourceFingerprint.toErrorResult()
 
+        SettingsPatch.addString("revanced_user_notice_message_signature_spoofing",
```

Comment:
Unless the user uses the switch, it will be unclear what the switch does. Usually, you'd use those dialogs to confirm the action taken or give a heads-up, not as a substitute to describe what the switch does.